deepmd/pt/model/descriptor/se_a.py

```
@@ -12,6 +13,11 @@
 
 import numpy as np
 import torch
+import torch.nn.functional as F
+
+from deepmd.pt.cxx_op import (
+    ENABLE_CUSTOMIZED_OP,
```

Ruff

`deepmd.pt.cxx_op.ENABLE_CUSTOMIZED_OP` imported but unused

Remove unused import: `deepmd.pt.cxx_op.ENABLE_CUSTOMIZED_OP`

(F401)